### DISCUSSION TITLE
Roborock S5 max custom room?

### DISCUSSION THREAD

**Discussion by Trilis29** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#630] [Q&A]
Hello, I wanted to create custom room. Because my robot shows I have max count. So I though making outline and putting coordinates to clean segment would work, but seems not. Is it possible to do something this way? 

Or I should just create many rectangles?

**Comment by PiotrMachowski** [comment]
Segment cleaning can use only already existing rooms (it starts cleaning using IDs, the outline is just for presentation purposes), so you have to use zone cleaning (rectangles)
